frontend/src/common/enums/index.ts

```diff
@@ -6,4 +6,8 @@ export * from './register';
 export * from './user';
 export * from './api';
 export * from './auth';
+export * from './register';
+export * from './users';
+export * from './api';
+export * from './edit-user';
```

Comment:
```suggestion
export * from './user';
export * from './api';
```
Move it all to the `user` folder